assessments/README.md

```diff
@@ -1,61 +1,40 @@
-that it meets its unique commitments to its own users and stakeholders.
-In open source, simply finding security-related information can be a very
-time consuming part of the the process. The process is designed to enable improved discovery of security information & streamlined security reviews in multiple ways:
+     Security reviews are a necessary, time intensive process. Each company, organization and project must perform its own reviews to ensure that it meets its unique commitments to its own users and stakeholders.
+     In open source, simply finding security-related information can be overwhelmingly difficult and a time consuming part of the security review. The CNCF security assessment, hereafter "security assessment," process is intended to enable improved discovery of security information & assist in streamlining internal and external security reviews in multiple ways:
 
    * Consistent documentation reduces review time
    * Established baseline of security-relevant information reduces Q&A
-   * Clear rubric for security profile enables organizations to align their
-   risk profile with project’s risk profile and effectively allocate resources
-   (for review and needed project contribution)
+   * Clear rubric for security profile enables organizations to align their risk profile with the project’s risk profile and effectively allocate resources (for review and needed project contribution)
    * Structured metadata allows for navigation, grouping and cross-linking
 
-We expect that this process will raise awareness of how specific open source
-projects affect the security of a cloud native system; however, separate
-activities may be needed to achieve that purpose using materials generated by
-the assessements.
+     We expect that this process will raise awareness of how specific open source projects affect the security of a cloud native system; however, separate activities may be needed to achieve that purpose using materials generated by the assessements.
 
 ## Outcome
 
-Each project assessment will:
+     Each project's security assessment will:
 1. ensure a clear description of the project's design goals with respect to
 security
-2. uncover design flaws and document known limitations
-3. document next steps toward increasing security of the project itself and/or
-increasing the applications of the project toward increasing security of the
-cloud native ecosystem
-
-Due to the nature and timeframe for the analysis, *this review is not meant
-to subsume the need for a professional security audit of the code*.  Audits
-of implementation vulnerabilities and similar issues at that level are not
-intended to be covered by this assessment.  The purpose of this effort is to
-uncover design flaws and to obtain a clear articulation of what the project's
-design goals and security properties are intended to be.
+2. uncover design and configuration flaws while documenting known limitations
```

Comment:
well, we won't "uncover design and configuration flaws" necessarily :)  

suggestion:
line 28 "... will include description of:"
1. the project's design goals with respect to
security
2. any aspects of design and configuration that could introduce risk
3. known limitations, such as expectations that some aspects of security will be handled by dependencies or complementary software
4. ext steps toward increasing security of the project itself and/or increasing the applications of the project toward a more secure cloud native ecosystem

(3) is a little wordy, but felt like "known limitations" was a separate point and thought it needed elaborating on a little bit

....

Reply:
added